### DISCUSSION TITLE
airflow 2.0.0 Feature List

### DISCUSSION THREAD

**Discussion by lixiaoyong12** [apache/airflow#13485] [Q&A] [answered]
I feel that 2.0.0 has changed a lot. Where can I find the feature list?

**Comment by potiuk** [comment, marked as answer]
https://github.com/apache/airflow/releases/tag/2.0.0

**Reply by lixiaoyong12** [reply]
Thank you very much， and i have another question  for you： I'd like to schedule future task in Dag(dag.schedule_interval is not None and allow_trigger_in_future=True）now.
